osu.Game/Skinning/SkinManager.cs

```diff
@@ -44,6 +44,11 @@ public class SkinManager : ModelManager<SkinInfo>, ISkinSource, IStorageResource
         /// </summary>
         public Skin DefaultClassicSkin { get; }
 
+        /// <summary>
+        /// Presents imported skin(s) to a user
+        /// </summary>
+        public Action<IEnumerable<Live<SkinInfo>>> PresentSkinsImport { get; set; }
```

Comment:
This is really weirdly circular. `skinImporter.PresentImport` invokes this, but then setting `PresentImport` on `SkinManager`... overwrites that? There should be no need for this.

Reply:
Wait, `SkinManager` have the `PresentImport` property? I didn't know about it

Reply:
https://github.com/ppy/osu/blob/679df9bf8b18e7b9760765f7e0a28ad4278c4528/osu.Game/Skinning/SkinManager.cs#L281-L284